glide-core/src/client/value_conversion.rs

```diff
@@ -1051,6 +1052,127 @@ pub(crate) fn get_value_type<'a>(value: &Value) -> &'a str {
 mod tests {
     use super::*;
 
+    #[test]
+    fn convert_xinfo_groups_xinfo_consumers() {
+        assert!(matches!(
+            expected_type_for_cmd(redis::cmd("XINFO").arg("GROUPS").arg("key")),
+            Some(ExpectedReturnType::ArrayOfMaps(&None))
+        ));
+
+        assert!(matches!(
+            expected_type_for_cmd(redis::cmd("XINFO").arg("CONSUMERS").arg("key").arg("group")),
+            Some(ExpectedReturnType::ArrayOfMaps(&None))
+        ));
+
+        // The format of the XINFO GROUPS and XINFO CONSUMERS responses are essentially the same, so we only need to
```

Comment:
split this out into a separate test please...
in the above, you are testing that XINFO GROUPS and CONSUMERS uses the expected type. 

In the following test, you are testing that the expected type properly converts data for that type.